lndr.admin.inc

```diff
@@ -1,48 +1,66 @@
 <?php
+
 /**
  * @file
- * Code for the Lndr module
+ * Code for the Lndr administration pages.
  */
 
-
-
 /**
- * Form builder callback
+ * Form builder callback.
  */
 function lndr_admin_settings($form, &$form_state) {
-
   $form['lndr_token'] = array(
     '#type' => 'textfield',
     '#title' => t('API token'),
     '#default_value' => variable_get('lndr_token', ''),
-    '#description'   => t("Your Lndr account API token, you can find this information under your user profile in Lndr"),
+    '#description' => t("Your Lndr account API token, you can find this information under your user profile in Lndr"),
     '#required' => TRUE,
   );
 
-  $form['lndr_debug_mode'] = array(
-    '#type' => 'checkbox',
-    '#title' => t('Developer mode'),
-    '#default_value' => variable_get('lndr_debug_mode', 0),
-    '#description' => t('Turn this on to test Lndr API data from local source instead of executing real API call'),
+  if (module_exists('lndr_debug')) {
```

Comment:
I think the module dependency here is not correct. Since the lndr_test.json exist in lndr_example module, should the dependency be lndr_example or both modules?

Reply:
I think you're right. Also, we do not use anywhere a `lndr_example_service_debug` function and/or a `lndr_service_debug` variable throughout the code.
I will remove the `lndr_debug` module and fix dependencies.